tests/merch.test.ts

```diff
@@ -564,4 +569,81 @@ describe('merch order pickup events', () => {
     expect(persistedPickupEvent.orders).toHaveLength(1);
     expect(persistedPickupEvent.orders[0]).toStrictEqual(persistedOrder);
   });
+
+  test('placing an order with a pickup event that has reached the order limit fails', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake({
+      points: 100,
+    });
+    const item = MerchFactory.fakeItem({
+      hidden: false,
+      monthlyLimit: 100,
+    });
```

Comment:
nit: I don't believe we need to create this merch item, since we aren't using it / asserting on it anywhere in this test. `PortalState.createMerchItemOption()` should take care of creating the item and collection for that option, so we can exclude this factory call.

Reply:
Resolved in 895837e.

Reply:
Apparently, removing the `fakeItem` call makes `fakeOption` generate an item that can potentially be hidden, which in turn causes unexpected failures in testing.

For now, I'll reverse this change, but perhaps we can adjust `fakeOption` to not hide items by default.